### ISSUE TITLE
Unable to compile ESPHelperFS.h

### ISSUE THREAD

**Issue by qinghuangchan** [ItKindaWorks/ESPHelper#22]
Error compiling 

`In file included from /home/qinghuang.chan/Arduino/libraries/ESPHelper/src/ESPHelper.cpp:25:0:
/home/qinghuang.chan/Arduino/libraries/ESPHelper/src/ESPHelperFS.h:93:10: error: 'StaticJsonBuffer' does not name a type
   static StaticJsonBuffer<JSON_SIZE> *_tmpBufPtr;
          ^
exit status 1
Error compiling for board Generic ESP8266 Module`

**Comment by qinghuangchan** [comment]
turns out i didnt link the libraries properly. sorry!

**Comment by GoranLilja** [comment]
Hey @qinghuangchan,
I'm having the same issue. Would you mind explaining how you solved this? If you remember, that is.
Thanks!

**Comment by GoranLilja** [comment]
Okay, I solved it. On [ArduinoJson's website](https://arduinojson.org/v5/doc/decoding/), they have a warning in which they state that Arduino Library Manager installs the latest version by default, which is a beta version. I simply downgraded the library, and it worked as expected.

1. `Sketch` → `Include Library` → `Manage Libraries…`.
2. Search for `ArduinoJson`.
3. Click to highlight the library, then select version `5.13.4` _(which is the latest non-beta version at the time of writing)_ and click <kbd>Install</kbd>.
